src/pages/markup.html

```diff
@@ -0,0 +1,272 @@
+{% extends '_layout.html' %}
+
+{% block title %} {{ super() }} | Markup {% endblock %}
+
+{% block content %}
+
+<img src="http://placehold.it/1000x250" height="250">
+
+<h1>Hello</h1>
```

Comment:
This markup file is kind of a mess - mainly just seeing how different elements would look. I also wasn't sure if you intended on showing styles through the markdown pages (style ramp post).